lib/ahoy/tracker.rb

```diff
@@ -154,6 +155,10 @@ def reset_visit
       delete_cookie("ahoy_track")
     end
 
+    def visit_from_param?
```

Comment:
looks like this one not used anywhere?

Reply:
@anatolmykolenko no, it is used. It's just a new public api which we use from within CreditNinja app:
https://github.com/KMDPartners/Mono/pull/4439/files#diff-f7057ca49f98982be32e620d68515aa0441302268df6b8c7f9630ce0ed6e193fR134